eng/Versions.props

```diff
@@ -19,12 +19,13 @@
     <MinorVersion>0</MinorVersion>
     <PatchVersion>0</PatchVersion>
     <PreReleaseVersionLabel>preview</PreReleaseVersionLabel>
-    <PreReleaseVersionIteration>1</PreReleaseVersionIteration>
+    <PreReleaseVersionIteration>5</PreReleaseVersionIteration>
     <!--
         When StabilizePackageVersion is set to 'true', this branch will produce stable outputs for 'Shipping' packages
     -->
     <StabilizePackageVersion Condition="'$(StabilizePackageVersion)' == ''">false</StabilizePackageVersion>
     <DotNetFinalVersionKind Condition="'$(StabilizePackageVersion)' == 'true'">release</DotNetFinalVersionKind>
+    <DefaultNetFxTargetFramework>net472</DefaultNetFxTargetFramework>
     <DefaultNetCoreTargetFramework>net7.0</DefaultNetCoreTargetFramework>
```

Comment:
Is this going to be right framework for the razor compiler? IIRC we had to revert this?

Reply:
We had to revert it due to an MSBuild mismatch. I can't find the thread right now, but I found out what was causing it, so we can revert my revert of your stuff when this goes through too.